cfme/tests/v2v/test_csv_import.py

```diff
@@ -0,0 +1,182 @@
+import fauxfactory
+import pytest
+import tempfile
+
+from cfme.fixtures.v2v import _form_data
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_VERSION
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.generators import random_vm_name
+from cfme.utils.wait import wait_for
+
+pytestmark = [
+    pytest.mark.ignore_stream('5.8'),
+    pytest.mark.provider(
+        classes=[RHEVMProvider],
+        selector=ONE_PER_VERSION
+    ),
+    pytest.mark.provider(
+        classes=[VMwareProvider],
+        selector=ONE_PER_VERSION,
+        fixture_name='second_provider'
+    )
+]
+
+
+@pytest.fixture(scope="function")
+def ims_map(appliance, v2v_providers):
+    """Fixture to create infrastructure mapping"""
+    form_data = _form_data(v2v_providers[0], v2v_providers[1])
+    return appliance.collections.v2v_mappings.create(form_data)
+
+
+def ims_plan(appliance, ims_map, csv=False):
+    """Function to create migration plan and select csv import option"""
+    plan_name = "map_{}".format(fauxfactory.gen_alpha(10))
+    plan_obj = appliance.collections.v2v_plans
+    view = navigate_to(plan_obj, 'Add', wait_for_view=True)
+    view.general.fill({
+        'infra_map': ims_map.name,
+        'name': plan_name,
+        'description': fauxfactory.gen_alpha(20)
+    })
+    if not csv:
```

Comment:
**Suggestion**We can skip this `csv` parameter, if I am not wrong this test is gonna deal with only `csv` based plans, so why to care for `non-csv` based plans?

Reply:
No, this csv parameter does not deals with plans, so it is for csv_import navigation option in UI.
For some function for example `valid_vm`, we don't need to go in csv import option so we are skipping that with this flag. If we don't have this flag, we have to add csv import selection code every other function.

Reply:
Makes sense.